CHANGELOG.md

```diff
@@ -16,5 +44,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Move OCS calls to app framework - consumes less resources - [#98](https://github.com/owncloud/notifications/pull/98)
 - Don't use escaped message for browser notification - [#100](https://github.com/owncloud/notifications/pull/100)
 
-[Unreleased]: https://github.com/owncloud/core/compare/v10.0.2...stable10
+[Unreleased]: https://github.com/owncloud/notifications/compare/v10.0.10...stable10
```

Comment:
This will need adjusting, to be `v10.0.10...master`
(and do we have a `v10.1.0` and `v10.1.1` tag here in `notifications` ? )